upgrade/rendering-changes/3-6-0.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 3.6.0
+description: Handle the rendering changes in the 3.6.0 release of the Telerik UI for Blazor components.
+page_title: Rendering Changes in 3.6.0
+slug: changes-in-3-6-0
+position: 2
+---
+
+# Rendering Changes in 3.6.0
```

Comment:
General food for thought - which **additions** fall in the scope of the article? The idea is that we should not document our HTML rendering for every new feature.

Adding new classes to previously existing is probably fine.